src/seraphis_mocks/jamtis_mock_keys.h

```diff
@@ -81,3 +102,5 @@ void make_random_address_for_user(const jamtis_mock_keys &user_keys, JamtisDesti
 } //namespace mocks
 } //namespace jamtis
 } //namespace sp
+
+BLOB_SERIALIZER(crypto::x25519_secret_key);
```

Comment:
```suggestion
BLOB_SERIALIZER(crypto::x25519_secret_key);

```
Files needs a newline at the bottom.

Reply:
Also, it is not appropriate to have `BLOB_SERIALIZER(crypto::x25519_secret_key);` here. If it should be anywhere, it should be in `src/serialization/crypto.h`.